crates/icrate/CHANGELOG.md

```diff
@@ -11,20 +11,94 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## icrate Unreleased - YYYY-MM-DD
 
 ### Added
-* Added the `CoreAnimation` framework.
+* Added the following frameworks:
+  - `Accessibility`
+  - `AdServices`
+  - `AdSupport`
+  - `AutomaticAssessmentConfiguration`
+  - `Automator`
+  - `BackgroundAssets`
+  - `BackgroundTasks`
+  - `BusinessChat`
+  - `CallKit`
+  - `ClassKit`
+  - `CloudKit`
+  - `Contacts`
+  - `CoreLocation`
+  - `DataDetection`
+  - `DeviceCheck`
+  - `EventKit`
+  - `ExceptionHandling`
+  - `ExtensionKit`
+  - `ExternalAccessory`
+  - `FileProvider`
+  - `FileProviderUI`
+  - `GameController`
+  - `GameKit`
+  - `IdentityLookup`
+  - `InputMethodKit`
+  - `LinkPresentation`
+  - `LocalAuthentication`
+  - `LocalAuthenticationEmbeddedUI`
+  - `MailKit`
+  - `MapKit`
+  - `Metal`
+  - `MetalFX`
+  - `MetalKit`
+  - `OSAKit`
+  - `CoreAnimation` (also known as `QuartzCore`)
+  - `SoundAnalysis`
+  - `Speech`
+  - `StoreKit`
+  - `UniformTypeIdentifiers`
+  - `UserNotifications`
+  - `WebKit`
 * Updated the SDK version from XCode `14.0.1` to `14.2`.
   - See differences [here](https://sdk.news/macOS-13.0/).
+* Added `Foundation::MainThreadBound` helper struct for things that are only
+  accesible from the main thread.
+* Added `#[deprecated]` annotations to the autogenerated output.
+* Added disambiguation for duplicated methods (e.g. `NSThread::threadPriority`
+  vs. `NSThread::threadPriority_class`).
+* Properly implemented protocols for defined classes.
+
+### Changed
+* Cfg-gated static `ns_string!` functionality behind the
+  `unstable-static-nsstring` cargo feature.
+* Autogenerated method parameters are now in snake-case, to for better IDE
```

Comment:
Small typo here I think? Maybe just `snake-case, for better`?

Reply:
Thanks!